src/ReaderTaskEither.ts

```diff
@@ -502,6 +502,22 @@ export const chainFirst: <R, E, A, B>(
     )
   )
 
+/**
+ * Less strict version of [`chainFirst`](#chainFirst).
+ *
+ * @category Monad
+ * @since 2.8.0
+ */
+export const chainFirstW: <R, E, A, B>(
```

Comment:
Actually *all* `chainFirst` combinators can be implemented as aliases of `chainFirstW`.

Looks like we can add `chainFirstW` to

- `IOEither`
- `ReaderEither`
- `StateReaderTaskEither`

too